docs/roles/others.md

```diff
@@ -15,6 +15,15 @@ Install NodeJS and NPM.
 * **nodejs_version** : The version to install, currently supports 6.x, 5.x, 4.x, 0.12 and 0.10
 * **nodejs_distro** : Is automatically set to either 'jessie' or 'wheezy' based on available information, you can also put an Ubuntu codename here.
 
+# OpenLDAP
+
+Install an OpenLDAP (slapd) server
```

Comment:
Add some useful info here: on what port can I reach it, using which credentials, etc.